app/assets/stylesheets/views/_brexit_checker_account_signup.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-experiment > .govuk-inset-text {
```

Comment:
We try to avoid the `>` selector as it reduces flexibility when it comes to changes to the markup. Would `.govuk-experiment .govuk-inset-text` work?

Reply:
I'll give it a go now!

Reply:
Yep that looks good:
![image](https://user-images.githubusercontent.com/3694062/94152256-45904b80-fe73-11ea-9eb5-23b3f8bd7e40.png)

![image](https://user-images.githubusercontent.com/3694062/94152963-16c6a500-fe74-11ea-96d9-f635521a4555.png)

Thanks for the suggestion!